src/stores/FreshNewStore.ts

```diff
@@ -222,6 +222,28 @@ const useFreshDataStore = create<FreshDataStore>()((set, get) => ({
       }));
     },
 
+    userUpdatesColumnIsPartOf(columnID, termId) {
+      set((state) => ({
+        columns: produce(state.columns, (draft) => {
+          if (termId) {
+            draft[columnID].isPartOf = termId;
+          } else {
+            delete draft[columnID].isPartOf;
+          }
+        }),
+      }));
+    },
+
+    userUpdatesMultiColumnMeasureCards(termId, isCollection) {
```

Comment:
```suggestion
    updateCollection(collectionTermId, isCollection) {
```
Inverse of above: store shouldn't care whether you show collections in cards or lists. Intent should be "make/destroy collection"

Related: should we make two actions here? One for "create collection" (like current but just setting to true), and "destroy collection", which not only sets isCollection to false, but also unmaps all columns currently pointing at collection before it does so?